### PR TITLE
Added BAG v1 fixtures

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: src/api/types.ts [hunk 1]
```diff
@@ -1,15 +1,28 @@
+/* eslint-disable camelcase */
 export interface Links {
```
Comment (jonkoops) [inline src/api/types.ts:2]: I believe we already have an existing type for hypermedia links in the root of the `api` dir.

File: src/api/types.ts [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
 }
 
-export interface APIReference {
-  count: number
+export interface SmallAPIReference {
+  count?: number
   href: string
 }
 
 export interface Geometrie {
```
Comment (jonkoops) [inline src/api/types.ts:13]: This can be replaced with `Geometry` from the `geojson` package.

File: src/api/v1/bag/ligplaats/types.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable camelcase */
+import { SmallAPIReference, APIReference, Geometrie } from '../../../types'
+
+export interface Root extends APIReference {
+  ligplaatsidentificatie: string
+  date_modified: string
+  document_mutatie: string
+  document_nummer: string
+  begin_geldigheid: string
+  einde_geldigheid: any
```
Comment (jonkoops) [inline src/api/v1/bag/ligplaats/types.ts:10]: Why do we have `any` as a type here?

**Review by jonkoops — APPROVED** [review]

File: src/api/v1/bag/ligplaats/types.ts (hunk 3 above)
Comment (chrisvanmook) [reply src/api/v1/bag/ligplaats/types.ts:10]: Value in the json is `null`

**Pull request merged**